fastparquet/converted_types.py

```diff
@@ -115,6 +115,7 @@ def convert(data, se):
     elif ctype == parquet_thrift.ConvertedType.INTERVAL:
         # for those that understand, output is month, day, ms
         # maybe should convert to timedelta
+        # TODO: seems like a np.view should do this much faster
```

Comment:
This is a special type I haven't seen yet. Given TIME_MILLIS and DATE, I don't know why anyone would want to use this different kind of compound interval.